[J159] update state 변경

GodDrinkTeJAVA opened the pull request:
기존의 state update는 params를 통해서 한 개씩 지정하는 방식으로 사용되었다.
또한, 값을 받으면 이를 toggle된 값으로 주는 방식을 사용하였었다.
하지만 이 경우 Mark as를 이용한 일괄 수정이 어렵다는 문제점이 있었다.
그러므로 이를 변경하여 array를 받으며,
이를 일괄적으로 해당 값으로 변경하는 방식으로 수정하였다.

GodDrinkTeJAVA committed:
[FIX] update state 변경

기존의 state update는 params를 통해서 한 개씩 지정하는 방식으로 사용되었다.
또한, 값을 받으면 이를 toggle된 값으로 주는 방식을 사용하였었다.
하지만 이 경우 Mark as를 이용한 일괄 수정이 어렵다는 문제점이 있었다.
그러므로 이를 변경하여 array를 받으며,
이를 일괄적으로 해당 값으로 변경하는 방식으로 수정하였다.

hyh1016 commented on server/middlewares/issue.js:
```diff
@@ -17,6 +17,14 @@ const createIssue = async (req, res) => {
 
+const updateIssueState = async (req, res) => {
+  const { isOpen, ids } = req.body;
+  console.log(req.body);
```
console..!

hyh1016 commented on server/models/query.js:
```diff
@@ -20,6 +20,7 @@ module.exports = {
 
   READ_ALL_ISSUE: 'select issue.id, issue.title as issue_title, issue.is_open, issue.user_id, changed_at, milestone_id, username, milestone.title as milestone_title from (issue join user on issue.user_id = user.id) left outer join milestone on issue.milestone_id = milestone.id;',
   CREATE_ISSUE: 'insert into issue(title, is_open, user_id, milestone_id, changed_at) values(?, ?, ?, ?, ?);',
+  UPDATE_ISSUE_STATE: 'update issue set is_open = ? where id in (?)',
```
세미콜론(;)...!

hyh1016 commented on server/middlewares/issue.js:
```diff
@@ -17,6 +17,14 @@ const createIssue = async (req, res) => {
 };
 
+const updateIssueState = async (req, res) => {
+  const { isOpen, ids } = req.body;
```
ids는 무슨 의미일지 여쭤봐도 될까요...?

hyh1016 reviewed (commented):
😎👍

GodDrinkTeJAVA committed:
[FIX] update state 관련 수정

update와 관련된 오타 및 일관성을 수정하였다.
1. toggle milestone의 작동 방식을 바꾸었다.
이전에는 false를 주면 true로 바꾸고, true를 false로 바꾸었지만
issue가 주어진 isOpen값으로 바꾸도록 수정되었기 때문에
이에 맞추어 isOpen값으로 주어지도록 변경하였다.
단, params로 id를 전달하는 방식은 유지되었다.
이름 또한 이에 맞추어 toggle에서 update로 바꾸었다.
2. console log 등 자잘한 잘못 수정
sql query semicolon, console log 등을 변경하였다.
